pvacseq/lib/pvacseq_utils.py

```diff
@@ -19,3 +23,50 @@ def iedb_to_prediction_method_lookup_dict():
 def iedb_prediction_methods():
     prediction_method_lookup_dict = iedb_to_prediction_method_lookup_dict()
     return sorted(prediction_method_lookup_dict.keys())
+
+def valid_allele_names_for_method(iedb_method):
+    #Ultimately we probably want this method to call out to IEDB but their command is currently broken
+    #curl --data "method=ann&species=human" http://tools-api.iedb.org/tools_api/mhci/
+    base_dir               = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+    iedb_alleles_dir       = os.path.join(base_dir, 'iedb_alleles')
+    iedb_alleles_file_name = os.path.join(iedb_alleles_dir, "%s.tsv" % iedb_method)
+    alleles = set()
+    with open(iedb_alleles_file_name) as iedb_alleles_file:
```

Comment:
It seems like, depending on the size of this list and the frequency of the calls to it, it may make sense to store it after we've read it from a file? Especially once it becomes a call to an external web service. If I'm reading it correctly, won't [this](https://github.com/griffithlab/pVAC-Seq/pull/96/files#diff-afe07e29b20ef4c68d244768d2dbc888R25) sequence of lines result in reading in all these files (or eventually making web calls) three different times?

Its also entirely probable I've missed something here and that's not the case :)